api/tests/emails_views_tests.py

```diff
@@ -21,6 +21,29 @@
 )
 
 
+@pytest.fixture
+def settings_without_sqlcommenter(settings: SettingsWrapper) -> SettingsWrapper:
+    """
+    Remove the sqlcommenter from the middleware.
+
+    For sqlite, it injects two queries into the recorded queries. The
+    first is a plain string, the second is the expected dictionary format.
+    This breaks the tests using django_assert_num_queries
+    First query: "SELECT id, ...
+    Second query: {"sql": "SELECT id, ..."}A
+    """
+
+    # The sqlcommenter middleware records queries twice for sqlite
```

Comment:
suggestion (non-blocking): this comment is redundant after the doc-block above
```suggestion
    # The sqlcommenter middleware records queries twice for sqlite
```